webpack.config.js

```diff
@@ -34,6 +34,7 @@ const isDevelopment = bundleEnv === 'development';
 const shouldMinify = process.env.hasOwnProperty( 'MINIFY_JS' )
 	? process.env.MINIFY_JS === 'true'
 	: ! isDevelopment;
+const commitSha = process.env.hasOwnProperty( 'COMMIT_SHA' ) ? process.env.COMMIT_SHA : '(unknown)';
```

Comment:
why don't we grab the commit sha using `spawnSync` in this file instead of in the docker pre-build step?

Reply:
Because `webpack` will be run inside the Docker container, and the `.git` directory isn't copied into the Docker container. So at this point there's no `git` repo.